### ISSUE TITLE
Radio group in Toolbar

### ISSUE THREAD

**Issue by sophieH29** [microsoft/fluent-ui-react#1470]
Based on [WAI-ARIA spec](https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html), radio group in Toolbar should follow next requirements:
- the active element should have set `aria-checked=true`
- up/down arrow keys should allow circular navigation within a radio group
